http_service/bugbug_http/models.py

```diff
@@ -110,6 +111,65 @@ def classify_bug(model_name: str, bug_ids: Sequence[int], bugzilla_token: str) -
     return "OK"
 
 
+def classify_issue(
+    model_name: str, owner: str, repo: str, issue_nums: Sequence[int]
+) -> str:
+    from bugbug_http.app import JobInfo
+
+    issue_ids_set = set(map(int, issue_nums))
+
+    issues = {}
```

Comment:
Nit: you can rewrite this with a dict comprehension to avoid multiple updates:
```Python
issues = {
    issue_num: github.fetch_issue_by_number(owner, repo, issue_num, True)
    for issue_num in issue_nums
}
```